settings.gradle.kts

```diff
@@ -344,16 +345,34 @@ gradle.rootProject {
         outputFile = layout.projectDirectory.file("architecture/platforms.md")
         elements = provider { architectureElements.map { it.build() } }
     }
-    tasks.register("platformsData", GeneratePlatformsDataTask::class) {
```

Comment:
It's kind of weird to do all of this in settings. Is there something that prevents us from doing it in the build.gradle.kts in the root itself?